Casks/m/moradownloader.rb

```diff
@@ -10,6 +10,14 @@
   livecheck do
     url "https://mora.jp/contents/data/system/noticeTouch.json"
     regex(/Mac.*?v?[._-](\d+(?:\.\d+)+)/i)
```

Comment:
```suggestion
    regex(/Mac.*?v?\.?(\d+(?:\.\d+)+)/i)
```

While we're here, we should make the delimiter after `v` optional. The text we're matching is like `Mac版：v.2.0.0.6` but the regex would fail to match `v2.0.0.6` or simply `2.0.0.6`. This is also an uncommon format, so it's probably fine to just use `\.` instead of `[._-]`.